src/NuGet.Core/NuGet.Protocol/Model/IPackageSearchMetadata.cs

```diff
@@ -31,6 +31,7 @@ public interface IPackageSearchMetadata
         string Summary { get; }
         string Tags { get; }
         string Title { get; }
+        bool IsRecommended { get; set; }
```

Comment:
This is incorrect. 

We should not change any NuGet.Core/* types for this change. 

This type maps to a server model, and recommendations are not related to servers.

Reply:
Removed.